frontend/src/components/Products/index.tsx

```diff
@@ -1,17 +1,18 @@
 import { h } from 'preact';
 import { useEffect, useState } from 'preact/hooks';
-import { Button } from '../Button/index';
+import { Button } from '../Button';
 import { getProductsData } from '../../services/api';
+import { CartService } from '../../services/cartService';
 
 export const Products = ({ activeSubIdCategory }) => {
   const [products, setProducts] = useState(null);
 
-  const handleAddToCart = () => {};
+  const handleAddToCart = (product) => {
+    CartService.addItem(product);
+  };
 
   useEffect(() => {
-    getProductsData(activeSubIdCategory).then((data) => {
-      data && setProducts(data.products);
-    });
+    getProductsData(activeSubIdCategory).then((data) => data && setProducts(data.products));
```

Comment:
you could use data?.products (optional-chaining) instead of data && data.products